tasks/deploy/config/base/morpho-blue-cbeth-usdc-182day.ts

```diff
@@ -0,0 +1,137 @@
+import {
+    Address,
+    encodeAbiParameters,
+    keccak256,
+    parseEther,
+    toHex,
+    zeroAddress,
+} from "viem";
+import {
+    CBETH_ADDRESS_BASE,
+    HyperdriveInstanceConfig,
+    SIX_MONTHS,
+    USDC_ADDRESS_BASE,
+    getLinkerDetails,
+    normalizeFee,
+    parseDuration,
+    toBytes32,
+} from "../../lib";
+import { BASE_FACTORY_NAME } from "./factory";
+import { BASE_MORPHO_BLUE_COORDINATOR_NAME } from "./morpho-blue-coordinator";
+
+export const BASE_MORPHO_BLUE_CBETH_USDC_182DAY_NAME =
+    "ElementDAO 182 Day Morpho Blue cbETH/USDC Hyperdrive";
+
+// USDC only has 6 decimals.
+const CONTRIBUTION = 100_000_000n;
+
+const morphoBlueParameters = encodeAbiParameters(
+    [
+        {
+            components: [
+                {
+                    name: "morpho",
+                    type: "address",
+                },
+                {
+                    name: "collateralToken",
+                    type: "address",
+                },
+                {
+                    name: "oracle",
+                    type: "address",
+                },
+                {
+                    name: "irm",
+                    type: "address",
+                },
+                {
+                    name: "lltv",
+                    type: "uint256",
+                },
+            ],
+            name: "MorphoBlueParams",
+            type: "tuple",
+        },
+    ],
+    [
+        {
+            morpho: "0xBBBBBbbBBb9cC5e90e3b3Af64bdAF62C37EEFFCb" as `0x${string}`,
+            collateralToken: CBETH_ADDRESS_BASE,
+            oracle: "0xb40d93F44411D8C09aD17d7F88195eF9b05cCD96" as `0x${string}`,
+            irm: "0x46415998764C29aB2a25CbeA6254146D50D22687" as `0x${string}`,
+            lltv: BigInt("860000000000000000"),
+        },
+    ],
+);
+
+export const BASE_MORPHO_BLUE_CBETH_USDC_182DAY: HyperdriveInstanceConfig<"MorphoBlue"> =
+    {
+        name: BASE_MORPHO_BLUE_CBETH_USDC_182DAY_NAME,
+        prefix: "MorphoBlue",
+        coordinatorAddress: async (hre) =>
+            hre.hyperdriveDeploy.deployments.byName(
+                BASE_MORPHO_BLUE_COORDINATOR_NAME,
+            ).address,
+        deploymentId: keccak256(toHex(BASE_MORPHO_BLUE_CBETH_USDC_182DAY_NAME)),
+        salt: toBytes32("0x42080085"),
+        extraData: morphoBlueParameters,
+        contribution: CONTRIBUTION,
+        // NOTE: The latest variable rate on the Morpho Blue market is 2.93% APY:
+        // https://app.morpho.org/market?id=0x1c21c59df9db44bf6f645d854ee710a8ca17b479451447e9f56758aee10a2fad&network=base&morphoPrice=0.75
+        fixedAPR: parseEther("0.0293"),
+        timestretchAPR: parseEther("0.05"),
```

Comment:
i'd be more comfortable with a 7.5% timstretchAPR. wdyt?

Reply:
We talked offline about sticking with 5%. The rationale is:

- Better for FRB since the liquidity requirements are lower.
- The market's variable APY has never gone above 13%. 
- We haven't seen Morpho markets sustain more than 20% APY for more than a couple of days.